<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeedRepairDialog</class>
 <widget class="QDialog" name="NeedRepairDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报修</string>
  </property>
  <widget class="QLabel" name="txtKind">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxKind">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>家具</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>墙面/地面</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>燃气</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtProblem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtProblem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtHouse">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>房号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxHouse">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>boxKind</tabstop>
  <tabstop>boxHouse</tabstop>
  <tabstop>edtProblem</tabstop>
  <tabstop>btnSubmit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnSubmit</sender>
   <signal>clicked()</signal>
   <receiver>NeedRepairDialog</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submit()</slot>
 </slots>
</ui>
